lua/telescope/builtin/__files.lua

```diff
@@ -410,7 +410,14 @@ files.current_buffer_fuzzy_find = function(opts)
       end,
     })
     for id, node in query:iter_captures(root, opts.bufnr, 0, -1) do
-      local hl = highlighter_query:_get_hl_from_capture(id)
+      local hl
+      -- update to changes on Neovim master, see https://github.com/neovim/neovim/pull/19931
+      -- TODO(clason): remove when dropping support for Neovim 0.7
+      if vim.fn.has "nvim-0.8" == 1 then
```

Comment:
we could store that eval in an local outside of the for loop so we dont check that every time.

Reply:
True -- although then I'd probably make it global so it can be used in other compat checks?

Reply:
And/or refactor this into a local `get_hl_from_capture_id` function?